pylint/testutils/reporter_for_tests.py

```diff
@@ -8,7 +8,7 @@
 from pylint.reporters import BaseReporter
 
 
-class TestReporter(BaseReporter):
+class ReporterForTests(BaseReporter):
```

Comment:
I don't see the point in renaming this class. Other classes of this module have their names ending with `Reporter` and this is no more the case here. Is it wise? Moreover i feel it much less 'english'

Reply:
I tried to rename because pytest consider class that start by `Test` to be test classes and complain about the `__init__` function. Do you have a better name ? Or maybe we can revert ?

Reply:
I understand the problem with pytest. Why not something along `GenericTestReporter`? or `GeneralTestReporter`?

Reply:
Clearly a better name, thanks :)